conda-env/conda_env/cli/main_export.py

```diff
@@ -62,6 +62,13 @@ def configure_parser(sub_parsers):
         help='Remove build specification from dependencies'
     )
 
+    p.add_argument(
+        '--skip-channel-prefix',
```

Comment:
One thing I meant to mention... I'm going to change this option name to `--ignore-channels`.  I'd prefer not to overload/confuse the word `prefix` in the conda context.